src/app.ts

```diff
@@ -49,18 +61,32 @@ export async function appFactory (options: AppOptions): Promise<{ app: Applicati
   app.configure(express.rest())
   app.configure(socketio())
 
-  // Authenticatoin service
+  // Authentication service
   app.configure(authentication)
 
   // Custom general services
   app.configure(sequelize)
   app.configure(configureStore)
-  app.configure(blockchain)
+  await waitForConfigure(app, blockchain)
```

Comment:
hmm... as I understand it depends on the order

Reply:
Yeah, it is not ideal, but the Web3Events needs to be initialized (its models have to be added to Sequelize and synced) before we can start initialization of the services...